presto-product-tests/src/main/java/com/facebook/presto/tests/cli/PrestoCliTests.java

```diff
@@ -52,9 +53,37 @@
     private final List<String> nationTableBatchLines;
 
     @Inject
-    @Named("databases.presto.server_address")
+    @Named("databases.presto.cli_server_address")
     private String serverAddress;
 
+    @Inject(optional = true)
```

Comment:
why optional?
Maybe we can force it to be either optional or not depending on list of tests to be executed?